plugins/quickinspector/quickinspector.cpp

```diff
@@ -110,7 +110,6 @@ Q_DECLARE_METATYPE(QSGNode::Flags)
 Q_DECLARE_METATYPE(QSGNode::DirtyState)
 Q_DECLARE_METATYPE(QSGGeometry *)
 Q_DECLARE_METATYPE(QMatrix4x4 *)
-Q_DECLARE_METATYPE(const QMatrix4x4 *)
```

Comment:
unrelated?

Reply:
this was previously causing a multiple declaration but doesnt appear to be doing so now, i can revert this.

Reply:
reverted